.github/workflows/release.yaml

```diff
@@ -144,17 +144,39 @@ jobs:
         run: |
           unset RELEASE_MINISIGN_PRIVATE_KEY
           unset RELEASE_MINISIGN_PRIVATE_KEY_PASSWORD
-      - name: Set FILES variable from tag
-        run: |
-          echo "FILES=$(find ${{github.workspace}}/.build/release/buf/assets -type f)" >> $GITHUB_ENV
       - name: Release
         id: ghr
         uses: softprops/action-gh-release@v1
+        env:
+          PATH: ${{github.workspace}}/.build/release/buf/assets
```

Comment:
I'd recommend using something like `ASSETS_PATH` here instead of `PATH` (not sure if that would break command execution).

Reply:
resolved in [e9ced1e](https://github.com/bufbuild/buf/pull/1657/commits/e9ced1e469041d3661458e71b4999b6f58e3d265)